Environment/FilesystemLibrary.hpp

```diff
@@ -15,4 +15,6 @@ class FilesystemLibrary final : public EnvironmentLibrary {
 
 public:
     void register_environment(lua_State *L) override;
+
+   static std::string location;
```

Comment:
Static variables should be only accessible through getters and setters for consistency.